database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,16 +14,21 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->bigIncrements('id');
+            $table->unsignedBigInteger('id')->autoIncrement();
+           // $table->primary('id');
             $table->string('name');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
```

Comment:
usually you can not edit old migrations, only create the new one with schema changes.